src/component/utils/Modal.tsx

```diff
@@ -35,44 +42,39 @@ const Modal = ({ isOpen, onCloseModal, children, size }: ModalProps) => {
     return candidate.includes(size) ? size : "basic";
   };
 
-  const onClickBackdrop = e => {
+  const closeModal = (e: MouseEvent<HTMLElement>) => {
     e.preventDefault();
     onCloseModal();
   };
 
   useEffect(() => {}, []);
 
-  const onClickContainer = e => {
-    // 이벤트 버블링 방지
+  const preventEventBubbling = (e: MouseEvent<HTMLElement>) => {
     e.stopPropagation();
   };
-  // 선언된 위치가 아닌 지정된 위치에 생성
-  const modalPosition = document.getElementById("modal");
 
-  return createPortal(
-    <>
-      <div
-        className="modal__backdrop"
-        onClick={onClickBackdrop}
-        aria-hidden="true"
-      >
+  const modalPosition = document.getElementById("portal");
+
+  return (
+    modalPosition &&
+    createPortal(
+      <>
         <div
-          className={`modal__container ${modalSize()}`}
-          onClick={onClickContainer}
-          onKeyPress={onClickContainer}
+          className="modal__backdrop"
+          onClick={closeModal}
```

Comment:
```suggestion
          onClick={closeModal}
          onKeyDown={closeModalWithESC}
```
ESC 이벤트를 `onKeyDown`에서 처리하면 어떨까요?

Reply:
https://github.com/jiphyeonjeon-42/frontend/pull/425/files#r1183275747

여기서 언급했듯 해당 기능은 포커싱에 상관없이 작동하길 기대하고 있어서
div의 onKeyDown보다는 window 객체의 이벤트가 더 적절하다고 생각합니다!